build/patches/JIT-less-toggle.patch

```diff
@@ -107,13 +107,15 @@ diff --git a/content/public/android/java/src/org/chromium/content/browser/Device
  import org.chromium.base.StrictModeContext;
  import org.chromium.content_public.common.ContentSwitches;
  import org.chromium.ui.base.DeviceFormFactor;
-@@ -22,4 +23,9 @@ public class DeviceUtilsImpl {
+@@ -22,4 +23,11 @@ public class DeviceUtilsImpl {
              }
          }
      }
 +    public static void addJitlessSwitch() {
-+	if (ContextUtils.getAppSharedPreferences().getBoolean("force_no_jit", false)) {
-+	    CommandLine.getInstance().appendSwitchWithValue("js-flags", "--jitless");
++	try (StrictModeContext ignored = StrictModeContext.allowDiskReads()) {
```

Comment:
@uazo why does it need to allow disk reads here? Does this introduce security holes?

Reply:
no, no problem, a few lines above it is the same thing (in the chromium code).